### PR TITLE
update pr-data.csv 

### PR DESCRIPTION
**Pull request by CaseyPan**
- `org.apache.shardingsphere.elasticjob.lite.integrate.disable.OneOffDisabledJobIntegrateTest.assertJobRunning` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.integrate.disable.ScheduleDisabledJobIntegrateTest.assertJobRunning` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.integrate.enable.OneOffEnabledJobIntegrateTest.assertJobInit` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.integrate.enable.ScheduleEnabledJobIntegrateTest.assertJobInit` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertListenException` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertMonitorWithDumpCommand` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertPortInvalid` DeveloperFixed (Added an issue regarding MovedorRenamed too)
- `org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceEnableTest.assertMonitorWithCommand` DeveloperFixed (Added an issue regarding MovedorRenamed too)

The tests has been fixed by developer on commit: 
054571c6d7caca455418e0f83056a0182315bd75
https://github.com/apache/shardingsphere-elasticjob/commit/054571c6d7caca455418e0f83056a0182315bd75
The log is on vm:
/home/ycp2/logs_iDflakies/shardingshpere-elastic-job/2d337265d (the commit before the fix)
/home/ycp2/logs_iDflakies/shardingshpere-elastic-job/054571c6d (the commit that fix it)

The tests are also moved to
test/e2e/src/test/java/org/apache/shardingsphere/elasticjob/test/e2e/snapshot/SnapshotServiceEnableE2ETest.java
On commit:
621cf953cf0382a652145eaf6d72df767f8766c4
[apache/shardingsphere-elasticjob@621cf95](https://github.com/apache/shardingsphere-elasticjob/commit/621cf953cf0382a652145eaf6d72df767f8766c4)

### CODE REVIEW

**Comment by darko-marinov** [comment]
As Zijie said, don't remove old info but use `;` to separate links. Also please resolve conflicts.

**Comment by CaseyPan** [comment]
@zzjas 
I have restored the old links. 
After running iDFlakies, I have checked that the renamed tests are no longer OD.

**Comment by darko-marinov** [comment]
To see that there are conflicts, look at the bottom of this page https://github.com/TestingResearchIllinois/idoft/pull/1018 It says `This branch has conflicts that must be resolved`.

To resolve the conflicts, rebase your work on top of the latest `main` from the original repo.

**Comment by CaseyPan** [comment]
@darko-marinov 
The current main `pr-data.csv` marked these tests as `DeveloperFixed` with a commit that doesn't seem to be the fix. 
These tests are renamed and moved to other places as I mentioned in the issues. Can I overwrite `DeveloperFixed` with `MovedOrRenamed`? How about the commit?

**Comment by darko-marinov** [comment]
Why do you think that the commit for `DeveloperFixed` is wrong? Did you run the tests before or after that commit?

**Comment by CaseyPan** [comment]
@darko-marinov  Yes. I ran the tests after the commit. Also, the commit I found the tests are moved and renamed is also later than the current one in pr-data.csv.

**Comment by darko-marinov** [comment]
You've changed one `Accepted` to `Opened`, but I don't understand why. That PR shows "Merged".

**Comment by CaseyPan** [comment]
I didn't notice the mistake! I've changed it back. @darko-marinov

**Comment by darko-marinov** [comment]
Thanks for cleaning this (but squash multiple commits into 1 for the next round). Do the tests remain `ID` after the move/rename?

**Comment by CaseyPan** [comment]
@darko-marinov No they didn't remain `OD` after the move/rename. Therefore I didn't create new entries.

**Comment by darko-marinov** [comment]
Were they `OD` before the rename? If not, then they were fixed, so find the commit for `DeveloperFixed`.

**Comment by CaseyPan** [comment]
@darko-marinov 
I see! Sorry I originally thought we should mark the newest commit related to the tests. But it makes sense that if it's fixed before we need to update the commit that fixes it.
So I reran iDFlakies on multiple commits and the commit in the original `pr-data.csv` is correct and it is indeed `DeveloperFixed`.
I have updated the `pr-data.csv`!

**Comment by darko-marinov** [comment]
Please squash 4 commits into 1.

Were are the logs showing that the tests were `OD` up to some commit and then fixed?

**Comment by CaseyPan** [comment]
@darko-marinov Commits squashed.
Logs can be found in my vm. It can also be verified by the last merged PR.
flaky-lists.json is too big to attached or mentioned in the PR description.
I would really want this PR to be merged.

File: pr-data.csv [hunk 1]
```diff
@@ -2821,14 +2821,14 @@ https://github.com/apache/shardingsphere-elasticjob,b022898ef1b8c984e17efb2a422e
...
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest.assertPortInvalid,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
-https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceEnableTest.assertMonitorWithCommand,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
+https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.integrate.disable.OneOffDisabledJobIntegrateTest.assertJobRunning,OD,DeveloperFixed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78;https://github.com/TestingResearchIllinois/idoft/issues/1012;https://github.com/apache/shardingsphere-elasticjob/commit/054571c6d7caca455418e0f83056a0182315bd75
```
Comment (darko-marinov) [inline pr-data.csv:2824]: The `Notes` includes https://github.com/TestingResearchIllinois/idoft/issues/1012 which describes where the test was moved or renamed, but this line claims `DeveloperFixed`. Do you have some logs for `DeveloperFixed`?

Comment (CaseyPan) [reply pr-data.csv:2824]: Updated the issues and also the PR description with log file path to my VM. Thanks!

**Comment by darko-marinov** [comment]
The best may be to meet right after the lecture to review these changes. I'm confused as the tests were already marked `DeveloperFixed`, and you're also marking them as `DeveloperFixed`.

**Comment by darko-marinov** [comment]
Thanks for the updates. I'll accept the PR, but please see my comments on the issue.

**Pull request merged**